cmd/ocm/gcp/gcp-client-shim.go

```diff
@@ -545,3 +545,12 @@ func (c *shim) createMemberRoleBindingForProject(
 		Role:    roleName,
 	})
 }
+
+// Checks for WIF config name or id in input
+func wifKeyArgCheck(args []string) error {
```

Comment:
This helper does not belong in this file as it does not relate to the shim. I would recommend placing it in a 'helpers.go' file.